frontend/app/src/constants/route.ts

```diff
@@ -9,4 +9,6 @@ export const ROUTE = {
   PROFILE: "/profile",
   PROFILE_USERID: "/profile/:userId",
   SETTINGS: "/settings",
+  QUESTIONS: "/questions",
+  QUESTION_ID: ":questionId",
```

Comment:
```suggestion
  QUESTION_ID: "/:questionId",
```
is the missing `/` intentional

Reply:
Yeah it is I think that's how react-router knows its a nested path vs absolute